KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaCuePointsManager.java

```diff
@@ -149,6 +153,15 @@ public void run() {
 							timer.schedule(timerTask, date);
 						}
 					}
+					logger.debug("Filtered list size: " + numFiltered);
+					//filter was empty
+					if (numFiltered == 0) {
+						return;
+					}
+					//update the filter:
+					if(periodic){
+						filter.updatedAtGreaterThanOrEqual = lastUpdatedCuePoint;
```

Comment:
sees very different then current date. I understand that there is no point in sending something before lastUpdatedCuePoint - but it seems to me the current date will always be greater